components/add_new_cv_form.tsx

```diff
@@ -86,27 +81,84 @@ export default function AddNewCvForm({ id }: Props) {
     }
 
     delete updatedCv.projects;
-    delete updatedCv.technical_skills;
+    delete updatedCv.certifications;
+    delete updatedCv.education;
 
     return supabase.from("cv").upsert(updatedCv).select();
   }
 
-  async function upsertSkills(values: any, cvData: any) {
-    values.cv_id = cvData[0].id;
-    return supabase.from("technical_skills").upsert(values);
+  async function upsertCertifications(certifications: any, cvId: string) {
+    if (certifications.length === 0) {
+      return await supabase.from("certifications").delete().eq("cv_id", cvId);
+    }
+
+    const updatedCertifications = certifications.map((certification: any) => ({
+      ...certification,
+      cv_id: cvId,
+    }));
+    return supabase.from("certifications").upsert(updatedCertifications);
+  }
+
+  async function upsertEducation(education: any, cvId: string) {
+    education.cv_id = cvId;
+    return supabase.from("education").upsert(education);
+  }
+
+  async function upsertProjects(projects: any, cvId: string) {
+    if (projects.length === 0) {
+      return await supabase.from("projects").delete().eq("cv_id", cvId);
+    }
+
+    const updatedProjects = projects.map((project: any) => {
+      const startDate = new Date(project.date_start);
+      startDate.setDate(15);
+
+      const endDate = new Date(project.date_end);
+      endDate.setDate(15);
+      return {
+        ...project,
+        cv_id: cvId,
+        date_start: startDate.toISOString(),
+        date_end: endDate.toISOString(),
+      };
```

Comment:
@ridjis 
We now have timestamp field in database, so we convert the date to UTC format and we change the day to 15 (14th) so we would not get any timezone issues.

Reply:
Just a more context for posterity — we've set date day to 15th, because we're only displaying local date and year in UI, so creating timestamp where only month and year is selected could display wrong local value if e.g. Feb 2022 (timestamp `2022-02-01T07:00:00.000Z`) will display Feb 2022 in Serbia, but will be Jan 2022 in Los Angeles for example, so setting date to middle of the month will ensure we're displaying correct month on UI.